api/node/__test__/eventloop.spec.mts

```diff
@@ -71,10 +71,35 @@ test.serial('quit event loop on last window closed', async (t) => {
     t.not(instance, null);
 
     instance.window().show();
-    await runEventLoop(() => {
-        setTimeout(() => {
-            instance.window().hide();
-        }, 2);
-    });
+    await runEventLoop(
+         () => {
+            setTimeout(() => {
+                instance.window().hide();
+            }, 2);
 
+    });
 })
+
+
+test.serial('quit event loop on last window closed with object', async (t) => {
+    let compiler = new private_api.ComponentCompiler;
+    let definition = compiler.buildFromSource(`
+
+    export component App inherits Window {
+        width: 300px;
+        height: 300px;
+    }`, "");
+    t.not(definition, null);
+
+    let instance = definition!.create() as any;
+    t.not(instance, null);
+
+    instance.window().show();
+    await runEventLoop({
+        runningCallback: () => {
+            setTimeout(() => {
+                instance.window().hide();
+            }, 2);
+        }
+    });
+})
```

Comment:
done

Reply:
Instead of removing this, another option is to create a separate .mts file - that'll work around the issue.

That said, it would be great to have a test for the functionality we're actually adding here: The ability to continue running the event loop even though the window is hidden. That is not tested here.